src/System.Linq.Expressions/tests/Unary/UnaryQuoteTests.cs

```diff
@@ -47,5 +48,461 @@ public void QuotePreservesTypingOfBlock(bool useInterpreter)
 
         public static Type Quote1(Expression<Action> e) => e.Body.Type;
         public static Type Quote2(Expression<Func<object>> e) => e.Body.Type;
+
+#if FEATURE_COMPILE
+        [Fact]
+        public static void Quote_Lambda_Action()
```

Comment:
Could this be `[Theory, ClassData(CompilationTypes)]` so it tests the same path for the interpreter? Or if it fails on interpreter then `[Theory, InlineData(false)]` so the a separate `[ActiveIssue]` test could pass in `true`?